iothub/service/src/Device.cs

```diff
@@ -109,9 +110,29 @@ public Device(string id)
         public virtual DeviceCapabilities Capabilities { get; set; }
 
         /// <summary>
-        /// Scope to which this device instance belongs to
+        /// The scope of the device. For edge devices, this is auto-generated and immutable. For leaf devices, set this to create child/parent
+        /// relationship.
         /// </summary>
+        /// <remarks>
+        /// For edge devices, the auto-generated device scope consists of a prefix "ms-azure-iot-edge://", the name of the parent edge device, a dash, and the
```

Comment:
I added this extra context to help users understand what this value should look like, and to know how do the right thing.